Library/Homebrew/cmd/info.rb

```diff
@@ -179,6 +179,43 @@ def info_formula(f)
 
     caveats = Caveats.new(f)
     ohai "Caveats", caveats.to_s unless caveats.empty?
+
+    output_analytics(f)
+  end
+
+  def output_analytics(f)
+    return if ENV["HOMEBREW_NO_ANALYTICS"]
```

Comment:
Should we automatically exclude the people that opt out of submitting analytics data ? Getting access to the data might not be equivalent to not wanting to share your own.